### PR TITLE
fix(ReportDubiousOwnership): Replace single quotes

### LINKED ISSUE #11767: Dubious repo ownership: Proposed command does not work for UNC paths
### Environment

- GitExtensions >= version: 4.0.1

### Issue description

Git proposes a command suitable for bash etc.
On Windows, the single quotes must be removed or rather be replaced with double quotes.

image

### Steps to reproduce

- try to open a repo in an UNC path or on a mapped network drive, (which always is normalized to the UNC path by git internally)
- click `Trust this repo` (added in #10436)

### Did this work in previous version of GitExtensions?

no

### Diagnostics

_No response_

### PR DESCRIPTION
**Pull request by mstv**
Fixes #11767

## Proposed changes

- `ReportDubiousOwnership`: Replace single quotes with double quotes in proposed git command

### Before

see issue

### After

image

## Test methodology 

- manual

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: src/app/GitUI/NBugReports/BugReportInvoker.cs [hunk 1]
```diff
@@ -366,6 +367,17 @@ static void ShowGitRepo(Form? ownerForm, string? workingDir)
                         });
                 }
             }
+
+            static string ReplaceQuotes(string command)
```
Comment (gerhardol) [inline src/app/GitUI/NBugReports/BugReportInvoker.cs:371]: Very specific usage...
```suggestion
            static string ReplacePrefixQuotes(string command)
```

File: src/app/GitUI/NBugReports/BugReportInvoker.cs [hunk 2]
```diff
@@ -366,6 +367,17 @@ static void ShowGitRepo(Form? ownerForm, string? workingDir)
...
+            {
+                if (!EnvUtils.RunningOnWindows() || !command.EndsWith('\''))
+                {
+                    return command;
+                }
+
+                int quoteIndex = command.IndexOf("'%(prefix)");
+                return quoteIndex < 0 ? command : @$"{command[..quoteIndex]}""{command[(quoteIndex + 1)..^1]}""";
```
Comment (gerhardol) [inline src/app/GitUI/NBugReports/BugReportInvoker.cs:379]: This should work, but I had to read a couple of times.
why not .StartsWith()?

**Review by gerhardol — APPROVED** [review]

File: src/app/GitUI/NBugReports/BugReportInvoker.cs (hunk 1 above)
Comment (RussKie) [inline src/app/GitUI/NBugReports/BugReportInvoker.cs:371]: A comment explaining what the method does with some examples would go a long way

**Review by RussKie — APPROVED** [review]
Modulo @gerhardol's comments

**Pull request merged**